tests/basic/pure_test.go

```diff
@@ -2880,6 +2863,196 @@ var _ = Describe("{FBDAMultiTenancyBasicTest}", func() {
 	})
 })
 
+var _ = Describe("{FBDAMultiTenancyUpdatePureNFSEnpoint}", func() {
+	var contexts []*scheduler.Context
+	var customConfigAppName string
+	var origCustomAppConfigs, customAppConfigs map[string]scheduler.AppConfig
+	testName := "fbda-mt-update-endp"
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("FBDAMultiTenancyUpdatePureNFSEnpoint", "Validate Pure NFS endpoint can be changed using pxctl", nil, 0)
+		Step("setup credential necessary for cloudsnap", createCloudsnapCredential)
+		customConfigAppName = skipTestIfNoRequiredCustomAppConfigFound()
+
+		// save the original custom app configs
+		origCustomAppConfigs = make(map[string]scheduler.AppConfig)
+		for appName, customAppConfig := range Inst().CustomAppConfig {
+			origCustomAppConfigs[appName] = customAppConfig
+		}
+
+		// update the custom app config with empty string for Pure NFS endpoint
+		// So that app uses NFS endpoint from pure.json secret.
+		Inst().AppList = []string{customConfigAppName}
```

Comment:
can you add logic to revert to original app_list . If we run this test along with other tests we might need to apps provided as part of the test run for other tests

Reply:
Removed modifing of AppList it self.